src/transformers/models/mistral/modeling_mistral.py

```diff
@@ -870,7 +870,7 @@ def _update_causal_mask(
         # to infer the attention mask.
 
         # cache_position must be valid here no matter which cache we use
-        past_seen_tokens = cache_position[0] if past_key_values is not None else 0
+        past_seen_tokens = past_key_values.get_seq_length() if past_key_values is not None else 0
```

Comment:
Same as in Llama, using `cache_position` is a dynamic control flow which is not supported currently by compile. The `fullgraph-compile` test fails without this change